dcpy/test/utils/test_versions.py

```diff
@@ -27,57 +39,51 @@ def test_sort_valid_versions(self):
         for version_list, sorted in [
             [
                 [
-                    versions.MajorMinor(year=2023, major=2),
-                    versions.MajorMinor(year=2023, major=1),
-                ],
-                [
-                    versions.MajorMinor(year=2023, major=1),
-                    versions.MajorMinor(year=2023, major=2),
-                ],
-            ],
-            [
-                [
-                    versions.Date(year=2023, month=1, day=2),
-                    versions.FirstOfMonth(year=2023, month=1),
-                ],
-                [
-                    versions.FirstOfMonth(year=2023, month=1),
-                    versions.Date(year=2023, month=1, day=2),
-                ],
-            ],
-            [
-                [
-                    versions.FirstOfMonth(year=2023, month=3),
-                    versions.FirstOfMonth(year=2023, month=2),
-                    versions.Date(year=2023, month=2, day=2),
+                    "23v2",
```

Comment:
Seemed nicer to work with strings here because they're more easily human-identifiable as "real" versions